ui/app/AppLayouts/Chat/CommunityComponents/CommunitiesPopup.qml

```diff
@@ -44,6 +44,7 @@ StatusModal {
 
     content: Column {
         width: popup.width
+        property alias searchBox: searchBox
```

Comment:
These changes here are unrelated to the changes described in the PR and can potentially moved into a separate commit.

This is a fixing a bug that was introduced when community modals were refactored. Let me know if this should live in a separate commit.